pkgs/development/python-modules/osmpythontools/default.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, beautifulsoup4
+, geojson
+, lxml
+, matplotlib
+, numpy
+, pandas
+, ujson
+, xarray
+}:
+
+buildPythonPackage rec {
+  pname = "osmpythontools";
+  version = "0.2.6";
+
+  src = fetchPypi {
+    pname = "OSMPythonTools";
+    inherit version;
+    sha256 = "efc72e3963971c6c7fd94bd374704a5b78eb6c07397a4ffb5f9176c1e4aee096";
+  };
+
+  propagatedBuildInputs = [
+    beautifulsoup4
+    geojson
+    lxml
+    matplotlib
+    numpy
+    pandas
+    ujson
+    xarray
+  ];
+
+  patches = [ ./remove-unused-dependency.patch ];
```

```suggestion
  patches = [ ./remove-unused-dependency.patch ];
  
  # no tests included
  doCheck = false;
 ```
this will also fix the "no setup.py could be found" issue
```

---

will eventually be fixed by https://github.com/NixOS/nixpkgs/pull/80595

---

How should we proceed here?

- Should `doCheck = false;` be included or not?
- Should we wait for #80595 (cd97c055a09c076de0cecf28f683d43d4da35d20 or b25621adf20d5c96a47f13f3df520bf6693bfff4) to hit `master` and then rebase #80444's branch on that?
  - if so, how and when will #80595 get into `master`?

---

it's in staging-next, for now I'm fine with `docheck = false;`

---

included `doCheck = false;` in [`5ccb2e15`](https://github.com/NixOS/nixpkgs/compare/e93faf67a877f2baa56dc1dfc532393d1002ad54..5ccb2e150d854791167e5a6e560017327821c1dc)